Dockerfile

```diff
@@ -163,6 +163,9 @@ RUN sed -r "s/^(.*)#.*$/\1/g" ${HOME}/datumaro/requirements.txt | xargs -n 1 -L
 RUN patch --binary -p1 < ${HOME}/cvat/apps/engine/static/engine/js/3rdparty.patch
 RUN chown -R ${USER}:${USER} .
 
+# Download OpenCV.js
+RUN wget https://docs.opencv.org/master/opencv.js -O ${HOME}/cvat/apps/engine/static/engine/js/3rdparty/opencv.js
```

Comment:
It is not a complete solution. For development locally we don't use docker. Thus it will be necessary to download/update the library manually. It is painful.

Reply:
Can you please share a docker free approach to use CVAT so I can try it locally?

Reply:
https://github.com/opencv/cvat/blob/develop/CONTRIBUTING.md